app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html

```diff
@@ -154,10 +150,8 @@ <h2 class="heading-medium" id="ifAgent">@messages("landingPage.ifAgent.h2")</h2>
     <h2 class="heading-medium" id="ifTrust">@messages("landingPage.ifTrust.h2")</h2>
 
     <p>@Html(messages("landingPage.ifTrust.p1", appConfig.trusteeResponsibilitiesUrl))</p>
-    @if(!indirectDisposalsEnabled | !mixedUseEnabled) {
-        <h2 class="heading-medium">@messages("landingPage.reportingDisposals.h2")</h2>
-        <p>@Html(messages("landingPage.reportingDisposals.p1", appConfig.nrcgtReturn))</p>
-    }
+    <h2 class="heading-medium">@messages("landingPage.reportingDisposals.h2")</h2>
```

Comment:
this shouldn't be here right? This was only being shown if any of the asset types were disabled but now they're always enabled